app/app.go

```diff
@@ -232,15 +236,16 @@ var (
 		govtypes.ModuleName:            {authtypes.Burner},
 		// liquiditytypes.ModuleName:      {authtypes.Minter, authtypes.Burner},
 		// ibctransfertypes.ModuleName: {authtypes.Minter, authtypes.Burner},
-		auctiontypes.ModuleName:     nil,
-		cdptypes.ModuleName:         {authtypes.Minter, authtypes.Burner},
-		cdptypes.LiquidatorMacc:     {authtypes.Minter, authtypes.Burner},
-		ununifidisttypes.ModuleName: {authtypes.Minter},
+		auctiontypes.ModuleName:            nil,
+		cdptypes.ModuleName:                {authtypes.Minter, authtypes.Burner},
+		cdptypes.LiquidatorMacc:            {authtypes.Minter, authtypes.Burner},
+		ecosystemincentivetypes.ModuleName: nil,
+		ununifidisttypes.ModuleName:        {authtypes.Minter},
 		// wasm.ModuleName:             {authtypes.Burner},
-		nft.ModuleName:               nil,
-		nftminttypes.ModuleName:      nil,
-		nftmarkettypes.ModuleName:    nil,
-		nftmarkettypes.NftTradingFee: nil,
+		nft.ModuleName:            nil,
+		nftminttypes.ModuleName:   nil,
+		nftmarkettypes.ModuleName: nil,
+		// nftmarkettypes.NftTradingFee: nil,
```

Comment:
Should this be removed or commented?

Reply:
what do you think?
cc: @mkXultra

Reply:
Hmm, if this module account is going to be used for later, I think it's fine to comment. Will need to get input from @mkXultra though